src/fsharp/FSharp.Build/Fsc.fs

```diff
@@ -253,7 +261,7 @@ type public Fsc () as this =
         
         match pathMap with
         | null -> ()
-        | _ -> builder.AppendSwitchIfNotNull("--pathmap:", pathMap.Split([|';'; ','|], StringSplitOptions.RemoveEmptyEntries), ",")
+        | _ -> builder.AppendSwitchIfNotNull("--pathmap:", pathMap |> splitAndWsTrim, ",")
```

Comment:
This changes behaviour. We specifically _don't_ want to split on space characters here as they could be used in a directory name.

Reply:
@saul, we don't split on space in splitWsTrim.  
````
            let array = s.Split([| ';'; ','; '\r'; '\n' |], StringSplitOptions.RemoveEmptyEntries)
````
There is a change in behaviour for disabledWarnings and warningsNotAsErrors for users who used spaced delimited properties.  I was okay with that since, space delimited lists isn't really a msbuild idiom, so I didn't think anyone would use it.  What do you think should I add it back?

Reply:
Pretend I ever said anything - I got confused by this:

```f#
let wsCharsToTrim = [| ' '; '\t'; '\"'; '\'' |]
```

That's what I get for trying to read PRs on mobile :)

Reply:
@saul , I promise I will never speak of this again, although, I too had to double check.